src/ucs/sys/topo.h

```diff
@@ -0,0 +1,87 @@
+/**
+* Copyright (C) NVIDIA Corporation. 2019.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifndef UCS_TOPO_H
+#define UCS_TOPO_H
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+
+#include <ucs/type/status.h>
+#include <limits.h>
+
+BEGIN_C_DECLS
+
+/** @file topo.h */
+
+/**
+ * @ingroup UCS_RESOURCE
+ * System Device abstraction
+ */
+typedef struct ucs_sys_device {
+    unsigned int id;        /**< Index of the unit */
```

Comment:
what does 'index' mean?

Reply:
Just an integer identifier associated with sys_device. I thought may be useful for search purposes instead of having perform multiple checks for match with name or bus_id.